dwave/embedding/chain_strength.py

```diff
@@ -37,10 +37,28 @@
 __all__ = ['uniform_torque_compensation', 'scaled']
 
 def uniform_torque_compensation(bqm, embedding=None, prefactor=1.414):
-    """Chain strength that attempts to compensate for torque that would break
-    the chain.
-
-    The RMS of the problem's quadratic biases is used for calculation.
+    r"""Chain strength that attempts to compensate for chain-breaking torque.
+
+    The root mean square (RMS) of the problem's quadratic biases is used to
+    calculate a value of chain strength that preforms reasonably well on many
+    problems.
+
+    As described in [Ray2020]_, optimal chain strength for a :term:`clique`
+    embedding is proportional to the variance of the coupling strength of the
+    chain's qubits, :math:`\propto \sigma^2 = \frac{2}{N(N-1)} \sum J_{ij}^2`,
+    where :math:`N` is the number of qubits and :math:`J_{ij}` the coupling
+    between qubits :math:`i` and :math:`j`.
+
+    Heuristically, this chain strength is also a good first guess for an
```

Comment:
For the logical ground state to be chain-intact after embedding it is also necessary that the chain strength be large enough that chain-intact states are favoured over chain broken ones. The dynamically-efficient scaling is also appropriate choice to meet this requirement in challenging models. Consider the following observation: